recipes/xtensor-zarr/meta.yaml

```
@@ -0,0 +1,56 @@
+{% set name = "xtensor-zarr" %}
+{% set version = "0.0.1" %}
+{% set sha256 = "51fb22c0701df3f9d41aa811db2dff034eb409fa6f83b35dfde9fe24fd26fec5" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://github.com/xtensor-stack/{{ name }}/archive/{{ version }}.tar.gz
+  sha256: {{ sha256 }}
```

```suggestion

package:
  name: {{ name|lower }}
  version: {{ version }}

source:
  url: https://github.com/xtensor-stack/{{ name }}/archive/{{ version }}.tar.gz
  sha256: 51fb22c0701df3f9d41aa811db2dff034eb409fa6f83b35dfde9fe24fd26fec5
```
This is slightly cleaner. I don't think we advertise the older style any more?